### DISCUSSION TITLE
How to add `--signoff` to gitbook commits?

### DISCUSSION THREAD

**Discussion by ElijahLynn** [GitbookIO/community#454] [1. How to]
We have a need to add a `--signoff` to our commits, how to add this with Gitbook automatically?

**Comment by aikoii** [comment]
Hi @ElijahLynn, thanks for your patience as we look into your case with DCO integration. I've also been in touch with your colleague and I'll keep you updated on any progress 🙏.

**Reply by ElijahLynn** [reply]
Thank you!
